src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1136,6 +1137,14 @@
   [cacrl :- schema/Str]
   (slurp cacrl))
 
+(schema/defn ^:always-validate
+  get-crl-last-modified :- org.joda.time.DateTime
```

Comment:
nit: once we've imported this above we can, and idiomatically do, refer to it as just `DateTime`.